client/components/App.jsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react'
-import { BrowserRouter as Router, Switch } from 'react-router-dom'
+
+import Login from './Login'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
 
 const App = () => {
   return (
     <Router>
+
```

Comment:
Please remove extra empty lines! throughout all the files.

Reply:
username changed to email and deleted the empty lines.